solr/packaging/test/test_status.bats

```diff
@@ -43,7 +43,6 @@ teardown() {
 @test "status shell script ignores passed in -solrUrl cli parameter from user" {
   solr start
   run solr status -solrUrl http://localhost:9999/solr
-  assert_output --partial "needn't include Solr's context-root"
```

Comment:
?  no longer germaine?   as in we're checking something that no longer needs checking because this is how it works?